### DISCUSSION TITLE
Calendar English  week days and years

### DISCUSSION THREAD

**Discussion by tintin10q** [youzan/vant#11447] [Q&A]
Hello. When I use the callendar component like this:

```
 <van-calendar
      title="Set expiration day"
      allow-same-day
      v-model:show="show_experiation_date"
      @confirm="onConfirmExperationDate"
      @cancel="show_experiation_date = false"
      confirm-text="`Choose this day`"
    />
```

I get this:

![image](https://user-images.githubusercontent.com/24190849/210220386-8fd9cd74-04fd-4dff-88c3-32da6a10dde2.png)

How can I have English week days like on your docs:  [https://vant-ui.github.io/vant/#/en-US/calendar](https://vant-ui.github.io/vant/#/en-US/calendar)?

**Comment by chenjiahan** [comment]
Vant supports multiple languages, you can use the Locale component to switch the languages.

Reference: https://vant-contrib.gitee.io/vant/#/en-US/locale

**Reply by xiadanxin** [reply]
This link cannot be opened.

**Reply by chenjiahan** [reply]
https://vant-ui.github.io/vant/#/en-US/locale

**Comment by xiadanxin** [comment]
Get, thanks!


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "youzan/vant"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月21日(星期三) 上午10:18
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [youzan/vant] Calendar English  week days and years (Discussion #11447)





 
https://vant-ui.github.io/vant/#/en-US/locale
